hknweb/candidate/views.py

```diff
@@ -19,6 +19,7 @@
 # decorators
 
 # used for things only officers and candidates can access
+# TODO: use permissions instead of just the groups
```

Comment:
If this is still a to-do, should we just do it this week before merging this pull request?

Reply:
This is for the future